geosnap/visualize/tsplot.py

```diff
@@ -0,0 +1,62 @@
+"""
+Visualization method for time series plots.
+"""
+
+import proplot as plot
+
+__all__ = ["tsplot"]
+
+"""
+Function for plotting timeseries data from community objects.
+
+Parameters
+----------
+community    : community object
+               community object
+column       : str
+               column to be graphed in a time series
+title        : str, optional
+               desired title of figure
+years        : list, optional
+               years to be graphed
+               default is every year in dataframe.
+scheme       : string,optional
+               matplotlib scheme to be used
+               default is 'quantiles'
+k            : int, optional
+               number of bins to graph.
+               Default is 5.
+save_fig     : boolean, optional
+               whether to save figure. Default is False.
+dpi          : int, optional
+               dpi of the saved image if save_fig=True
+               default is 500
+"""
+
+
+def tsplot(community, column, title='',
+           years=[], scheme='quantiles',
+           k=5,save_fig=False, dpi=500,
+           **kwargs):
+    if not years:
+        f, axs = plot.subplots(ncols=len(community.gdf.year.unique()))
+        for i, year in enumerate(sorted(community.gdf.year.unique())):  # sort to prevent graphing out of order
+            community.gdf[community.gdf.year == year].plot(column=column, ax=axs[i], scheme=scheme, k=k, **kwargs,
+                                                           legend=True, legend_kwds={'ncols': 1, "loc": "b"})
```

Comment:
lets also expose `legend_kwds`. A good way to do that is have the default value be None, then do an 
```
if not legend_kwds:
	legend_kdws={'ncols': 1, "loc": "b"}
```

Reply:
Done with your recommended approach.
https://github.com/spatialucr/geosnap/pull/243/commits/2735b637d325085be6e023ef114e59ead43264d8